### DISCUSSION TITLE
Lesson -3

### DISCUSSION THREAD

**Discussion by Captain1918** [Cyfrin/foundry-full-course-cu#1060] [Q&A]
Unable to connect my remix  IDE with metamask wallet.  When i click on injected provider option its showing message as
"No injected provider found. Make sure your provider (e.g. MetaMask, ...) is active and running (when recently activated you may have to reload the page)."
How to solve this issue

**Comment by Abhi-1631** [comment]
Do u have the metamask extension in the same browser you are using the remix ide with? Ideally when you click on the injected provider option it should popup the metamask and ask your confirmation for connecting to the remix

**Comment by Captain1918** [comment]
I have fixed it bro.Initially i was using downloaded remix  ide instead of
online editor.So it's not  metamask option.But now in online editor i am
getting metamask option .
